scripts/tests/test_validate_unwanted_patterns.py

```diff
@@ -0,0 +1,423 @@
+with pytest.raises(ValueError, match="foo"):
+    pass
+"""
+            ),
+            [],
+        ),
+        (
+            (
+                """
+# with pytest.raises(ValueError, match="foo"):
+#    pass
+"""
+            ),
+            [],
+        ),
+        (
+            (
+                """
+# with pytest.raises(ValueError):
+#    pass
+"""
+            ),
+            [],
+        ),
+        (
+            (
+                """
+with pytest.raises(
+    ValueError,
+    match="foo"
+):
+    pass
+"""
+            ),
+            [],
+        ),
+        (
+            (
+                """
+with pytest.raises(ValueError):
+    pass
+"""
+            ),
+            [
+                (
+                    1,
+                    (
+                        "Bare pytests raise have been found. "
+                        "Please pass in the argument 'match' "
+                        "as well the exception."
+                    ),
+                ),
+            ],
+        ),
+        (
+            (
+                """
+with pytest.raises(ValueError, match="foo"):
+    with pytest.raises(ValueError):
+        pass
+    pass
+"""
+            ),
+            [
+                (
+                    2,
+                    (
+                        "Bare pytests raise have been found. "
+                        "Please pass in the argument 'match' "
+                        "as well the exception."
+                    ),
+                ),
+            ],
+        ),
+        (
+            (
+                """
+with pytest.raises(ValueError):
+    with pytest.raises(ValueError, match="foo"):
+        pass
+    pass
+"""
+            ),
+            [
+                (
+                    1,
+                    (
+                        "Bare pytests raise have been found. "
+                        "Please pass in the argument 'match' "
+                        "as well the exception."
+                    ),
+                ),
+            ],
+        ),
+        (
+            (
+                """
+with pytest.raises(
+    ValueError
+):
+    pass
+"""
+            ),
+            [
+                (
+                    1,
+                    (
+                        "Bare pytests raise have been found. "
+                        "Please pass in the argument 'match' "
+                        "as well the exception."
+                    ),
+                ),
+            ],
+        ),
+        (
+            (
+                """
+with pytest.raises(
+    ValueError,
+    # match = "foo"
+):
+    pass
+"""
+            ),
+            [
+                (
+                    1,
+                    (
+                        "Bare pytests raise have been found. "
+                        "Please pass in the argument 'match' "
+                        "as well the exception."
+                    ),
+                ),
+            ],
+        ),
+    ],
+)
+def test_pytest_raises(data, expected):
```

Comment:
Can you split this into two tests where one is acceptable values and the other is a `_raises` test?

I'm hoping that will help limit the parametrization since you won't need to supply the error message if you do things that way. Right now the parametrization is pretty tough to read (partially due to black formatting rules)